File: leaf-code-generation/src/main/java/com/leaf/domain/TableInfo.java
package com.leaf.domain;

/**
 * @author jhlz
 * @version 0.0.1
 * @since 2023/9/10 14:57
 */
public record TableInfo(String tableName,
                        String filedName,
                        String fieldType,
                        String maxLen,
                        String comment,
                        String notnull,
                        String defaultValue) {
}

File: leaf-generator/src/main/kotlin/com/leaf/domain/TableInfo.kt
package com.leaf.domain

/**
 * @author jhlz
 * @version x.x.x
 */
data class TableInfo(
    /* 数据库名 */
    val tableName: String,
    /* 数据库 schema */
    val tableSchema: String,
    /* 类型 */
    val tableType: String,
    /* 备注 */
    val tableDesc: String,
)